src/tempo_cluster.py

```diff
@@ -173,6 +186,10 @@ class TempoClusterProviderAppData(DatabagModel):
 
     tempo_config: Dict[str, Any]
     loki_endpoints: Optional[Dict[str, str]] = None
+    ca_cert: Optional[str] = None
+    # plaintext cacert
```

Comment:
missing `server_cert: Optional[str] = None`

Reply:
right, it's not in sync with what's in the coordinator
will fix